### ISSUE TITLE
Please include SigMF support

### ISSUE THREAD

**Issue by loidmen** [jopohl/urh#1068]

**Comment by 777arc** [comment]
Agreed, even if it's in the most minimal form which is super simple, https://pysdr.org/content/iq_files.html#sigmf-and-annotating-iq-files
